src/Alert/Alert.js

```diff
@@ -9,55 +10,66 @@ export class Alert extends Component {
         };
     }
 
-  closeAlertHandler = () => {
-      this.setState({
-          isActive: false
-      });
-  };
+    closeAlertHandler = () => {
+        this.setState({
+            isActive: false
+        });
+    };
 
-  render() {
-      const {
-          type,
-          link,
-          linkText,
-          dismissable,
-          children,
-          className,
-          ...props
-      } = this.props;
-      return (
-          <div>
-              {this.state.isActive && (
-                  <div
-                      className={`fd-alert${dismissable ? ' fd-alert--dismissible' : ''}${
-                          type ? ' fd-alert--' + type : ''
-                      }${className ? ' ' + className : ''}`}
-                      role='alert'
-                      {...props}>
-                      {dismissable && (
-                          <button
-                              aria-controls='j2ALl423'
-                              aria-label='Close'
-                              className='fd-alert__close'
-                              onClick={() => this.closeAlertHandler()} />
-                      )}
-                      {children}
-                      {link && (
-                          <a className='fd-link' href={link}>
-                              {linkText}{' '}
-                              <span className='sap-icon--arrow-right sap-icon--s' />
-                          </a>
-                      )}
-                  </div>
-              )}
-          </div>
-      );
-  }
+    render() {
+        const {
+            type,
+            link,
+            linkText,
+            dismissible,
```

Comment:
you changed `dismissable` to `dismissible`

that is a breaking change and while we should allow for breaking changes while we are 0.X.X, we should not intermix that work with this PR.

Reply:
Sounds good. I saw that they were different and assumed they should match. I'll open an issue to fix instead.

Reply:
https://github.com/SAP/fundamental-react/issues/234